build.sbt

```diff
@@ -30,12 +31,19 @@ inThisBuild(Def.settings(
   ),
   testOptions += Tests.Argument(TestFrameworks.JUnit, "-q", "-v"),
   Dependencies.Versions,
-  Formatting.formatSettings
+  Formatting.formatSettings,
+  shellPrompt := { s => Project.extract(s).currentProject.id + " > " }
 ))
 
+val dontPublishSettings = Seq(
```

Comment:
The [`NoPublish`](https://github.com/akka/akka-http/blob/master/project/Publish.scala#L8) plugin achieves the same thing, so this should not be needed.